docs/install-upgrade/overview.md

```diff
@@ -5,159 +5,95 @@
-preparation and configuration.
+This section is dedicated to the Hedgehog Fabric installation on bare-metal control node(s) and switches, their
+preparation and configuration. To install the vlab see [Vlab Overview](../vlab/overview.md).
 
-Get `hhfab` installed following instructions from the [Download](../getting-started/download.md) section.
+Download and install `hhfab` following instructions from the [Download](../getting-started/download.md) section.
 
 The main steps to install Fabric are:
 
 1. Install `hhfab` on the machines with access to the Internet
     1. [Prepare Wiring Diagram](./build-wiring.md)
-    1. [Select Fabric Configuration](./config.md)
+    1. [Select Fabric Configuration](./config.md) // TODO - section on dhcp or ntp servers, the FAB.yaml
     1. [Build Control Node configuration and installer](#build-control-node-configuration-and-installer)
 1. [Install Control Node](#install-control-node)
-    1. Install Flatcar Linux on the Control Node
-    1. Upload and run Control Node installer on the Control Node
+    1. Insert USB with control-os image into Fabric Control Node
+    1. Boot the node off the USB to initiate the installation
+1. Prepare Management Network
+    1. Connect switch to Fabric control node
+    1. Connect 1GbE Management ports of switches to control switch
 1. Prepare supported switches
-    1. [Install Hedgehog ONIE (HONIE) on them](./onie-update.md)
-    1. Reboot them into ONIE Install Mode to have them automatically provisioned
+    1. Boot them into ONIE Install Mode to have them automatically provisioned
 
-## Build Control Node configuration and installer
+## Build Control Node configuration and Installer
+Hedgehog has created a command line utility, called `hhfab`, that will help generate the wiring diagram, validate the supplied configurations, and generate an installation image (.img) suitable for writing to a USB flash drive or mounting via IPMI virtual media.
 
-It's the only step that requires Internet access, to download artifacts and build the installer.
+### HHFAB commands to make a bootable image
+1. `hhfab init --wiring wiring-lab.yaml`
+1. The `init` command will generate a `fab.yaml` file, edit the `fab.yaml` file for your needs
+    1. ensure the correct boot disk (e.g. `/dev/sda`) and control node NIC names are supplied
+1. `hhfab validate`
+1. `hhfab build`
 
-Once you've prepared the Wiring Diagram, initialize Fabricator by running `hhfab init` command and passing optional
-configuration into it as well as wiring diagram file(s) as flags. Additionally, there are a lot of customizations
-available as flags, e.g. to setup default credentials, keys and etc. For more details on the command invocation,
-refer to `hhfab init --help`.
+The installer for the fabric will be generated in `$WORKDIR/result/`. This installation image is 7.5 GB in size. It is named control-1-usb.img. Once the image is created, it can be written to a USB drive, or mounted via virtual media.
```

Comment:
```suggestion
The installer for the fabric is generated in `$WORKDIR/result/`. This installation image is named `control-1-usb.img` and is 7.5 GB in size. Once the image is created, you can write it to a USB drive, or mount it via virtual media.
```

Is the size (7.5 GB) stable, or should we provide something more approximative (“about 8 GB”)?